portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -28,16 +28,27 @@
  */
 @WebServlet("/login-status")
 public class LoginStatusServlet extends HttpServlet {
+  private static UserService userService;
+  private static String loginUrl;
+  private static String logoutUrl;
+
+  @Override
+  public void init() {
+    userService = UserServiceFactory.getUserService();
+    logoutUrl = userService.createLogoutURL("/");
+    loginUrl = userService.createLoginURL("/");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    UserService userService = UserServiceFactory.getUserService();
+    response.setContentType("application/json");
 
     if (userService.isUserLoggedIn()) {
       String userEmail = userService.getCurrentUser().getEmail();
 
-      response.getWriter().println("<p>1 " + userEmail + "</p>");
+      response.getWriter().println("{\n  \"isLoggedIn\": true,\n  \"email\": \"" + userEmail + "\"\n}");
```

Comment:
I would consider using a json library to encode these results rather than resorting to manual string manipulation.

This way, any sort of typos would actually be caught by the compiler.

Reply:
You're right, I will change it.